plugins/polio/api.py

```diff
@@ -284,36 +284,64 @@ def loop_on_rounds(self: Any, rounds: Any) -> list:
                         month = round.started_at.month
                         row["rounds"] = {}
                         row["rounds"][str(month)] = []
-                        row["rounds"][str(month)].append(self.get_round(round))
+                        row["rounds"][str(month)].append(self.get_round(round, self))
                         data_row.append(row)
                     else:
                         row = [sub for sub in data_row if sub["country_id"] == round.campaign.country.id][0]
                         row_index = data_row.index(row)
                         if row is not None:
                             month = round.started_at.month
                             if str(month) in data_row[row_index]["rounds"]:
-                                data_row[row_index]["rounds"][str(month)].append(self.get_round(round))
+                                data_row[row_index]["rounds"][str(month)].append(self.get_round(round, self))
                             else:
                                 data_row[row_index]["rounds"][str(month)] = []
-                                data_row[row_index]["rounds"][str(month)].append(self.get_round(round))
-
+                                data_row[row_index]["rounds"][str(month)].append(self.get_round(round, self))
         return data_row
 
     @staticmethod
-    def get_round(round):
+    def get_round(round: Any, self: Any) -> Any:
```

Comment:
Ceci semble très bizarre ...

Soit c'est une staticmethod et il n'y a pas de notion de "self"

Pour rappel :

What is a static method?
Static methods, much like [class methods](https://www.programiz.com/python-programming/methods/built-in/classmethod), are methods that are bound to a class rather than its object.

Donc une méthode statique n'a pas de notion de "self" l'objet mais bien de d'une eventuelle classe

Dans ce cas-ci vu qu'il y a un @staticmethod, on peut appeler `CampaignViewSet.get_round(xxx)`(donc sur la classe et pas sur une instance de l'objet).

A mon avis ce que tu veux faire ici c'est de retirer le @staticmethod pour que cela devienne `def get_round(self, round):` et après quand tu l'appelle tu fais juste `self.get_round(round)` et automatiquement le premier paramètre sera l'objet appelant ? Est-ce que c'est compréhensible ?

Egalement mettre un typage `: Any` partout ou rien, il vaut mieux rien mettre car Any ne donne aucune information sur le type ...